### ISSUE TITLE
Question about MACM evaluation on MATH dataset

### ISSUE THREAD

**Issue by adityabasarkar** [bin123apple/MACM#4]
Hello!

I just wanted to ask about how you evaluated your agent. You said that you evaluated it against 1/3 of the MATH dataset. Did you mean 1/3 of just the testing or training partition in MATH dataset or 1/3 of the total 12500 problems in this dataset?

Thanks.

**Comment by bin123apple** [comment]
Hey, thanks for asking. 

We evaluated it by using $\frac{1}{3}$ problems from the whole dataset.  

We aimed to maintain the same distribution of levels and problem categories as in the original dataset. To achieve this, first, split all the problems into different sub-types based on their level and category, and then select $\frac{1}{3}$ from each sub-type.

One thing to note is that some problems prompt GPT's code interpreter to generate images. For these problems (~ $100$), I temporarily excluded them during testing.

I've been quite busy lately, but I plan to add the image recognition functionality to the Agent when I have some free time.
